decidim-core/app/views/layouts/decidim/_process_header.html.erb

```diff
@@ -21,9 +21,24 @@
       <div class="process-nav">
         <div class="row column process-nav__content">
           <ul>
+            <li>
+              <%= active_link_to decidim.participatory_process_path(current_participatory_process), active: :exact, class: "process-nav__link", class_active: "is-active" do %>
+                <%= icon "process" %>
+                <%= t ".process_menu_item" %>
+              <% end %>
+            </li>
+
             <% current_participatory_process.features.each do |feature| %>
               <li>
-                <%= link_to translated_attribute(feature.name), decidim.feature_path(current_participatory_process, feature), class: "process-nav__link" %>
+                <%= active_link_to decidim.feature_path(current_participatory_process, feature), class: "process-nav__link", active: :inclusive, class_active: "is-active" do %>
+                  <% if feature.manifest.icon %>
```

Comment:
What about moving this logic to `Feature`? Adding an `icon` method that has this logic.